DepositeCalcTests/Tests/CalculatorPageTests.cs

```diff
@@ -160,6 +160,20 @@ public void YearsFieldTests()
             // Assert
             Assert.AreEqual(expactedEndDate, calculatorPage.EndDate, "Incorrect value in the end date field");
         }
+
+        [Test]
+        public void SettingLnkTest()
+        {
+            // Arrange
+            var calculatorPage = new CalculatorPage(driver);
+            string expectedUrl = "https://localhost:5001/Settings";
+
+            // Act
+            calculatorPage.ClickOnSettingsLnk();
+
+            // Assert
+            Assert.AreEqual(expectedUrl, driver.Url, "Incorrect page");
+        }
```

Comment:
Let's rewrite this test, and you should add a boolean property `IsOpened` to the SettingsPage. It should return `true` if the page is opened (e.g. url ends with 'settings', or some specific controls are visible) and `false` in other case.

As a result, the test will check, that 'Settings page' is opened, not that 'URL' is changed.

```suggestion
        {
            // Arrange
            var calculatorPage = new CalculatorPage(driver);

            // Act
            var settingsPage = calculatorPage.OpenSettings();

            // Assert
            Assert.IsTrue(settingsPage.IsOpened);
        }
```

Reply:
Sorry @varajan but it's not clear for me =( Maybe you talking about the method in the SettingsPage? 
Something like this:

        public bool IsOpened()
        {
            return driver.Url.Contains("Settings");
        }

Reply:
Also we can apply a similar code for LogoutLinkTest (create a similar method on the LoginPage